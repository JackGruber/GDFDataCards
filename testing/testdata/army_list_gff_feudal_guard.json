{
    "id": "eU-WQZOdGpvb",
    "gameSystem": "gff",
    "armyId": "w15ncjbqsqk3tcet",
    "armyIds": [
        "w15ncjbqsqk3tcet"
    ],
    "armyFaction": null,
    "armyName": "Feudal Guard",
    "modified": "2024-10-27T15:48:03.989Z",
    "saveVersion": 3,
    "favourite": false,
    "armyVersions": [
        {
            "version": "3.2.2",
            "armyId": "w15ncjbqsqk3tcet"
        }
    ],
    "listPoints": 295,
    "list": {
        "id": "eU-WQZOdGpvb",
        "key": "iNgMmfrf1NUiY6ReKNDSg",
        "name": "Feudal Guard",
        "description": "",
        "pointsLimit": 300,
        "campaignMode": false,
        "narrativeMode": false,
        "forceOrg": true,
        "gameSystem": "gff",
        "isCloud": false,
        "modified": "2024-10-27T15:48:03.989Z",
        "cloudModified": null,
        "activationCount": 5,
        "modelCount": 7,
        "units": [
            {
                "id": "Tejdlmn",
                "armyId": "w15ncjbqsqk3tcet",
                "selectionId": "ZWoN5",
                "selectedUpgrades": [
                    {
                        "instanceId": "fL2j-t4rV",
                        "upgradeId": "hwwOKuH",
                        "optionId": "yyugx"
                    }
                ],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "Jw0Fw2y",
                "armyId": "w15ncjbqsqk3tcet",
                "selectionId": "4L1RL",
                "selectedUpgrades": [],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "FwOWLvi",
                "armyId": "w15ncjbqsqk3tcet",
                "selectionId": "JbnWS",
                "selectedUpgrades": [
                    {
                        "instanceId": "M9yjglKmW",
                        "upgradeId": "ylwIKLi",
                        "optionId": "tANsN-5"
                    },
                    {
                        "instanceId": "Zl9eZkyjn",
                        "upgradeId": "zRVAQ0X",
                        "optionId": "n1JY_dO"
                    }
                ],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "JWcThfe",
                "armyId": "w15ncjbqsqk3tcet",
                "selectionId": "haY4a",
                "selectedUpgrades": [
                    {
                        "instanceId": "mj7O-UHJi",
                        "upgradeId": "G3LOgRZ",
                        "optionId": "lIDxv9-"
                    },
                    {
                        "instanceId": "1CKDcGydf",
                        "upgradeId": "lujjvqa",
                        "optionId": "Pcvl5L3"
                    }
                ],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "d_dzOfJ",
                "armyId": "w15ncjbqsqk3tcet",
                "selectionId": "Z6lUh",
                "selectedUpgrades": [
                    {
                        "instanceId": "bo5ratdIe",
                        "upgradeId": "lL1tV-x",
                        "optionId": "o4uNh"
                    }
                ],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            }
        ]
    }
}