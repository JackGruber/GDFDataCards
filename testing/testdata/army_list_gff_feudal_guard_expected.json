{
    "armyId": "w15ncjbqsqk3tcet",
    "gameSystem": "gff",
    "gameSystemId": 3,
    "armyVersions": [
        {
            "version": "3.2.2",
            "armyId": "w15ncjbqsqk3tcet"
        }
    ],
    "armyName": "Feudal Guard",
    "listPoints": 295,
    "listName": "Feudal Guard",
    "units": [
        {
            "type": "Elite Captain",
            "name": "Elite Captain",
            "armyId": "w15ncjbqsqk3tcet",
            "id": "Tejdlmn",
            "cost": 60,
            "defense": 3,
            "quality": 4,
            "upgrades": [
                "C1",
                "9kMqi"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": "2",
                    "count": 1,
                    "name": "Cpt. Heavy Rifle",
                    "range": 24,
                    "specialRules": [],
                    "ap": 1
                },
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "CCW",
                    "specialRules": []
                }
            ],
            "equipment": [
                {
                    "name": "Paymaster",
                    "specialRules": [
                        {
                            "id": "bIZKJ",
                            "type": "ArmyBookRule",
                            "name": "Deep Pockets",
                            "label": "Deep Pockets"
                        }
                    ]
                }
            ],
            "rules": [
                {
                    "id": "2atvlmICQSCV",
                    "name": "Fearless",
                    "label": "Fearless"
                },
                {
                    "id": "T08VkBuVmNZ_",
                    "name": "Hero",
                    "label": "Hero"
                },
                {
                    "id": "a0YtInGiUDd6",
                    "rating": "3",
                    "name": "Tough",
                    "label": "Tough(3)"
                },
                {
                    "id": "bIZKJ",
                    "type": "ArmyBookRule",
                    "name": "Deep Pockets",
                    "label": "Deep Pockets"
                }
            ],
            "upgradeCost": [
                15
            ]
        },
        {
            "type": "Feudal Sniper",
            "name": "Feudal Sniper",
            "armyId": "w15ncjbqsqk3tcet",
            "id": "Jw0Fw2y",
            "cost": 40,
            "defense": 4,
            "quality": 5,
            "upgrades": [],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "Sniper Rifle",
                    "range": 30,
                    "specialRules": [
                        {
                            "type": "ArmyBookRule",
                            "id": "Zx4mWN0SbmK8",
                            "name": "Reliable",
                            "label": "Reliable"
                        }
                    ],
                    "ap": 1
                },
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "CCW",
                    "specialRules": []
                }
            ],
            "equipment": [],
            "rules": [
                {
                    "id": "2atvlmICQSCV",
                    "name": "Fearless",
                    "label": "Fearless"
                },
                {
                    "id": "FAMbscK5vhSu",
                    "name": "Scout",
                    "label": "Scout"
                },
                {
                    "id": "9jyvRoBODD-O",
                    "name": "Stealth",
                    "label": "Stealth"
                }
            ]
        },
        {
            "type": "Feudal Guardsmen",
            "name": "Feudal Guardsmen",
            "armyId": "w15ncjbqsqk3tcet",
            "id": "FwOWLvi",
            "cost": 45,
            "defense": 4,
            "quality": 5,
            "upgrades": [
                "A1",
                "D1",
                "E1"
            ],
            "size": 3,
            "notes": null,
            "weapons": [
                {
                    "attacks": 1,
                    "count": 2,
                    "name": "Rifle",
                    "range": 24,
                    "specialRules": []
                },
                {
                    "attacks": 1,
                    "count": 3,
                    "name": "CCW",
                    "specialRules": []
                },
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "Shred Rifle",
                    "range": 18,
                    "specialRules": [
                        {
                            "id": "cavWDboL4ubs",
                            "type": "ArmyBookRule",
                            "name": "Rending",
                            "label": "Rending"
                        }
                    ]
                }
            ],
            "equipment": [],
            "rules": [
                {
                    "id": "2atvlmICQSCV",
                    "name": "Fearless",
                    "label": "Fearless"
                },
                {
                    "id": "bLVZU4onzi2E",
                    "type": "ArmyBookRule",
                    "name": "Medical Training",
                    "label": "Medical Training",
                    "count": 1
                }
            ],
            "upgradeCost": [
                10,
                15
            ]
        },
        {
            "type": "Feudal OGRE",
            "name": "Feudal OGRE",
            "armyId": "w15ncjbqsqk3tcet",
            "id": "JWcThfe",
            "cost": 65,
            "defense": 3,
            "quality": 4,
            "upgrades": [
                "K1"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "Bash",
                    "specialRules": []
                },
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "Heavy Baton",
                    "specialRules": [],
                    "ap": 2
                }
            ],
            "equipment": [
                {
                    "name": "Combat Shields",
                    "specialRules": [
                        {
                            "id": "wsqfB0fq69eG",
                            "type": "ArmyBookRule",
                            "name": "Shield Wall",
                            "label": "Shield Wall"
                        }
                    ]
                }
            ],
            "rules": [
                {
                    "id": "2atvlmICQSCV",
                    "name": "Fearless",
                    "label": "Fearless"
                },
                {
                    "id": "JoOCVPx1xT1G",
                    "name": "Furious",
                    "label": "Furious"
                },
                {
                    "id": "a0YtInGiUDd6",
                    "rating": "3",
                    "name": "Tough",
                    "label": "Tough(3)"
                },
                {
                    "id": "wsqfB0fq69eG",
                    "type": "ArmyBookRule",
                    "name": "Shield Wall",
                    "label": "Shield Wall"
                }
            ],
            "upgradeCost": [
                5,
                5
            ]
        },
        {
            "type": "Feudal Specialist",
            "name": "Feudal Specialist",
            "armyId": "w15ncjbqsqk3tcet",
            "id": "d_dzOfJ",
            "cost": 30,
            "defense": 4,
            "quality": 5,
            "upgrades": [
                "G1"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "CCW",
                    "specialRules": []
                },
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "Flamer",
                    "range": 12,
                    "specialRules": [
                        {
                            "id": "w_vX0mi58KKt",
                            "type": "ArmyBookRule",
                            "rating": 3,
                            "name": "Blast",
                            "label": "Blast(3)"
                        },
                        {
                            "id": "Zx4mWN0SbmK8",
                            "type": "ArmyBookRule",
                            "name": "Reliable",
                            "label": "Reliable"
                        }
                    ]
                }
            ],
            "equipment": [],
            "rules": [
                {
                    "id": "2atvlmICQSCV",
                    "name": "Fearless",
                    "label": "Fearless"
                },
                {
                    "id": "H9E9PIWpGRdw",
                    "name": "Relentless",
                    "label": "Relentless"
                }
            ],
            "upgradeCost": [
                5
            ]
        }
    ]
}